.pre-commit-config.yaml

```diff
@@ -10,7 +10,7 @@ repos:
     files: ^(ahk/_async/.*\.py|\.unasync-rewrite\.py|tests/_async/.*\.py)
     pass_filenames: false
     additional_dependencies:
-        - unasync
+        - git+https://github.com/spyoungtech/unasync.git@unasync-remove
```

Comment:
Once https://github.com/python-trio/unasync/pull/75 is merged and the new unasync version is published to pypi, we can remove this change